connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -348,7 +347,7 @@ private void commitOffsets(long now, boolean closing) {
 
         if (commitableOffsets.equals(lastCommittedOffsets)) {
             log.debug("{} Skipping offset commit, no change since last commit", this);
-            onCommitCompleted(null, commitSeqno);
+            onCommitCompleted(null, commitSeqno, null);
```

Comment:
Yeah, I wasn't sure why that would be the case, but didn't want to change it if not necessary. I couldn't convince myself there was a situation where it was _wrong_.